data/scripts/sol-oic-gen.py

```diff
@@ -1488,18 +1493,30 @@ def = sol_oic_server_register_definition(defn_endpoint, resource_type,
     parser.add_argument("--node-type-impl",
                         help="Path to store the node type implementation",
                         required=True)
+    parser.add_argument("--node-type-gen-c",
+                        help="Relative path to source generated with "
+                        "sol-flow-node-type-gen for inclusion purposes.",
+                        required=True)
+    parser.add_argument("--node-type-gen-h",
+                        help="Relative path to header generated with "
+                        "sol-flow-node-type-gen for inclusion purposes.",
+                        required=True)
     args = parser.parse_args()
 
     def seems_schema(path):
         return path.endswith('.json') and (path.startswith('oic.r.') or path.startswith('core.'))
 
+    json_name = os.path.basename(args.node_type_json)
+    if (json_name[-5:] == ".json"):
```

Comment:
not a big deal, but in python  you should drop the parenthesis and you can use `json_name.endswith(".json")`